packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -374,7 +381,8 @@ class _ProductPageState extends State<NewProductPage> {
     );
   }
 
-  Widget? _buildAttributeChipForValidAttributes(Attribute attribute) {
+  Widget? _buildAttributeChipForValidAttributes(
+      Attribute attribute, bool returnNullIfStatusUnknown) {
```

Comment:
Consider making it a named argument for readability at the call site

Reply:
Sure. Done